VideoLocker/res/layout/row_course_outline_list.xml

```diff
@@ -4,8 +4,8 @@
     android:layout_width="match_parent"
     android:layout_height="64dp"
     android:paddingTop="4dp"
-    android:paddingBottom="4dp"
     android:orientation="vertical"
+    android:background="@drawable/course_list_item_selector"
```

Comment:
do we really need to do this refactoring ? Current implementation isn't that bad.